https-everywhere-checker

```diff
@@ -1 +1 @@
-Subproject commit 2ec94f1ce11a2046408a8df228447e2f0dd85cda
+Subproject commit ce35ca84ec75f3205b098d5de73ea293fc22f187
```

Comment:
Looks like you're essentially reverting https://github.com/EFForg/https-everywhere/commit/7b76e745566cff08c9638b0b7138e2d0d35199b6

Might want to git pull, then run git submodule update --init to make sure your locally checked out submodule is fully up to date. Not sure if you want to just then add a commit to this patch chain to revert this change, or resubmit the pull request